src/_includes/navbar.njk

```diff
@@ -86,11 +86,11 @@
           </li>
         {% endfor %}
       </ul>
-      <div class="absolute right-0">
+      {# <div class="absolute right-0">
```

Comment:
We need Get Tickets 😅  Dont remove. Also ensure it's coming up in mobile view as well.

Reply:
`Get tickets` is there in the navbar in mobile view, Is it fine ?
<img width="364" alt="Screenshot 2025-05-31 at 11 13 09 AM" src="https://github.com/user-attachments/assets/0230fad4-2c03-4c83-a4b0-6a2f5f755045" />

Reply:
Ah cool. Yes works then.